### DISCUSSION TITLE
How to use rederables with RichHandler()

### DISCUSSION THREAD

**Discussion by byt3bl33d3r** [Textualize/rich#2920] [Q&A]
Heya,

I'm trying to use `JSON()` with `RichHandler()` so i can highlight json when logging to the terminal. How does one use renderables when using `RichHandler()` with logging?

Thanks!~

**Comment by willmcgugan** [comment]
The logging module only works with strings, so you can't use renderables with the logging handler. You can display json with `console.log` which doesn't use the logging module.
